bin/fba_pipweights

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
+
+"""
+Example of how one could compute PIP weights.
+"""
+
+from mpi4py import MPI
+
+import os
+import sys
+
+import argparse
+
+from bitarray import bitarray
+
+import fitsio
+
+import fiberassign
+
+from fiberassign.utils import Logger, distribute_discrete
```

Comment:
`distribute_discrete` was added to utils and imported here, but doesn't appear to be used.  Is this a leftover of some other parallelism testing?